docs/core/runtime-config/garbage-collector.md

```diff
@@ -562,6 +564,41 @@ These configuration settings don't have specific MSBuild properties. However, yo
 > [!TIP]
 > If you're setting the option in *runtimeconfig.json*, specify a decimal value. If you're setting the option as an environment variable, specify a hexadecimal value. For example, to limit the heap usage to 30%, the values would be 30 for the JSON file and 0x1E or 1E for the environment variable.
 
+### Region range
+
+Starting with .NET 7.0 the GC heap switched its physical representation from segments to regions for 64-bit Windows and Linux. See [Maoni Stephens' blog article](https://itnext.io/how-segments-and-regions-differ-in-decommitting-memory-in-the-net-7-gc-68c58465ab5a) for more details. With this change the GC reserves a range of virtual memory during initialization. Note that this is only reserving memory, not committing (the GC heap size is committed memory). It's merely a range to define the maximum range the GC heap can commit. For most applications they don't need to commit nearly this much.
```

Comment:
It might be good to outline what this value could be reasonably set to. For instance, if the user thinks that their working set is max 1GB, is it reasonable to set this to 2 GB? How will changing the range affect the performance of the GC? If they don't know the working set of their app, is 2x physical memory a better pick?

Reply:
yes it'd definitely be good to mention! thanks for catching that. I would say between 2x and 5x (2x is if you have more than 256GB physical memory and 5x is what we do when you have hardlimit specified). I'll do an edit.